docs/guides/how_to_guides/creating_batches/how_to_load_a_database_table_view_or_query_result_as_a_batch.rst

```diff
@@ -5,134 +5,246 @@ How to load a database table, view, or query result as a batch
 
-        .. code-block:: python
+            - To manually build ``table`` or ``view``-based Batch Kwargs, create a dictionary:
 
-            batch_kwargs = {
-                "datasource": "my_db",
-                "schema": "my_schema",  # schema is optional; default schema will be used if it is omitted
-                "table": "my_view"  # note that the "table" key is used even to validate a view
-            }
+                .. code-block:: python
 
-    - To use a Batch Kwargs Generator, call the ``build_batch_kwargs`` method:
+                    batch_kwargs = {
+                        "datasource": "my_db",
+                        "schema": "my_schema",  # schema is optional; default schema will be used if it is omitted
+                        "table": "my_view"  # note that the "table" key is used even to validate a view
+                    }
 
-        For example, if your configured generator is called ``my_generator`` and you wish to access the ``my_view`` asset, you can call:
+            - To use a Batch Kwargs Generator, call the ``build_batch_kwargs`` method:
 
-        .. code-block:: python
+                For example, if your configured generator is called ``my_generator`` and you wish to access the ``my_view`` asset, you can call:
 
-            batch_kwargs = context.build_batch_kwargs("my_db", "my_generator", "my_view")
+                .. code-block:: python
 
-    - To manually build ``query``-based Batch Kwargs, create a dictionary:
+                    batch_kwargs = context.build_batch_kwargs("my_db", "my_generator", "my_view")
 
-        .. code-block:: python
+            - To manually build ``query``-based Batch Kwargs, create a dictionary:
 
-            batch_kwargs = {
-                "datasource": "my_db",
-                "query": "SELECT * FROM my_schema.my_table WHERE '1988-01-01' <= date AND date < '1989-01-01';
-            }
+                .. code-block:: python
 
-    - To use a Query Batch Kwargs Generator, call the ``build_batch_kwargs`` method:
+                    batch_kwargs = {
+                        "datasource": "my_db",
+                        "query": "SELECT * FROM my_schema.my_table WHERE '1988-01-01' <= date AND date < '1989-01-01';
+                    }
 
-        .. code-block:: python
+            - To use a Query Batch Kwargs Generator, call the ``build_batch_kwargs`` method:
 
-            batch_kwargs = context.build_batch_kwargs(
-                "my_db",
-                "queries",
-                "movies_by_date",
-                query_parameters={
-                    "start": "1988-01-01",
-                    "end": "1989-01-01"
-                }
-            )
+                .. code-block:: python
 
-#. **Obtain an Expectation Suite to use to validate your batch**.
+                    batch_kwargs = context.build_batch_kwargs(
+                        "my_db",
```

Comment:
Are the my_db, queries, movies_by_date keys *args? If not, could we write down what these are assigned to explicitly?